.github/workflows/age-daily-deploy.yml

```diff
@@ -3,7 +3,7 @@ name: Age Daily Deploy
 on:
   push:
     paths:
-      - age/**
+      - age/contracts/**
```

Comment:
Why is it here if it's only the daily part ?

Also true for the others.

Reply:
What do you mean? I didn't understand

Reply:
done